types/src/v0/v0_3/stake_table.rs

```diff
@@ -65,7 +65,7 @@ pub type IndexedStake = (
 );
 
 #[derive(Clone, derive_more::derive::Debug)]
-pub struct StakeTableFetcher {
```

Comment:
Don't need to update this but I wonder why the change. If in the code we use as `stake_table::Fetcher` then yes, the new name is better, but if not, the old name is more descriptive and not overly verbose, isn't it?

Reply:
I see.